Dockerfile

```diff
@@ -1,38 +1,17 @@
-FROM ubuntu:14.04
-MAINTAINER Colin Alston <colin@praekelt.com>
-RUN apt-get update && apt-get -y --force-yes install libjpeg-dev zlib1g-dev libxslt1-dev libpq-dev nginx redis-server supervisor python-dev python-pip
-RUN apt-get -y install libffi-dev gettext
-
-RUN pip install --upgrade pip
-
-ENV PROJECT_ROOT /deploy/
-ENV DJANGO_SETTINGS_MODULE tuneme.settings.docker
-
-WORKDIR /deploy/
-
-COPY tuneme /deploy/tuneme
-COPY polls /deploy/polls
-COPY locale /deploy/locale
-ADD manage.py /deploy/
-ADD requirements.txt /deploy/
-ADD setup.py /deploy/
-ADD README.rst /deploy/
-ADD VERSION /deploy/
-
+FROM praekeltfoundation/django-bootstrap
+COPY . /app
 RUN pip install -e .
```

Comment:
I'd put this `COPY` and `RUN` as late as possible so that everything before them is cached. They can go after the `ENV` commands.